src/scripts/scripts/zone/gruuls_lair/boss_high_king_maulgar.cpp

```diff
@@ -266,7 +266,7 @@ struct boss_olm_the_summonerAI : public BossAI
                 case EVENT_DEATH_COIL:
                 {
                     AddSpellToCast(SPELL_DEATH_COIL, CAST_RANDOM);
-                    events.ScheduleEvent(eventId, 20000);
+                    events.ScheduleEvent(eventId, 10000);
```

Comment:
Olm the Summoner will Death Coil the one who is having aggo. This will happen quite often, so make sure to have two tanks ready on him, to taunt once he Death Coils the other tank.

http://tacticalwow.awardspace.com/tactics/GruulsLair/